app/src/main/res/values-pcm/strings.xml

```diff
@@ -0,0 +1,511 @@
+  <string name="preferences">How you like am</string>
+  <string name="profile_progress_activity_title">Profile Progress Page</string>
+  <string name="search">Find-am</string>
+  <string name="fraction_error_invalid_chars">Abeg only use numerical digits, spaces or forward slashes (/)</string>
+  <string name="fraction_error_invalid_format">Abeg enter a valid fraction (e.g., 5/3 or 1 2/3)</string>
+  <string name="fraction_error_divide_by_zero">Abeg no put 0 for di denominator</string>
+  <string name="fraction_error_larger_than_seven_digits">None of di nombas for di fraction go get more dan 7 digits.</string>
+  <string name="number_error_starting_with_floating_point">Abeg start your ansa with nomba (e.g.,”0” for 0.5).</string>
+  <string name="number_error_invalid_format">Abeg put correct nomba.</string>
+  <string name="number_error_larger_than_fifteen_characters">Di ansa fit get at most 15 digits (0–9) or sign (. or -).</string>
+  <string name="ratio_error_invalid_chars">\n    Abeg write a ratio wey get nomba separated by colons (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_format">\n    Abge enter a valid ratio (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_colons">\n    Your ansa get two colons (:) next to each other.</string>
+  <string name="ratio_error_invalid_size">\n    Nomba of terms no dey equal to di required terms.</string>
+  <string name="ratio_error_includes_zero">\n    Ratios no suppose get 0 for inside.</string>
+  <string name="unknown_size">Size wey dey no know</string>
+  <string name="size_bytes">%s Bytes</string>
+  <string name="size_kb">%s KB</string>
+  <string name="size_mb">%s MB</string>
+  <string name="size_gb">%s GB</string>
+  <string name="correct">You get am!</string>
+  <string name="topic_prefix">Topic: %s</string>
+  <plurals name="chapter_count">
+    <item quantity="one">1 Chapter\n    </item>
+    <item quantity="other">\n      %s Chapters\n    </item>
+  </plurals>
+  <plurals name="story_count">
+    <item quantity="one">1 Story\n    </item>
+    <item quantity="other">\n      %s Stories\n    </item>
+  </plurals>
+  <plurals name="story_total_chapters">
+    <item quantity="one">%s of %s Chapter Completed</item>
+    <item quantity="other">%s of %s Chapters Completed</item>
+  </plurals>
+  <plurals name="lesson_count">
+    <item quantity="one">1 Lesson\n    </item>
+    <item quantity="other">\n      %s Lessons\n    </item>
+  </plurals>
+  <plurals name="completed_story_count">
+    <item quantity="one">1 Story Completed\n    </item>
+    <item quantity="zero">%s Stories Completed\n    </item>
+    <item quantity="other">\n      %s Stories Completed\n    </item>
+  </plurals>
+  <plurals name="ongoing_topic_count">
+    <item quantity="one">1 Topic in Progress\n    </item>
+    <item quantity="zero">%s Topics in Progress\n    </item>
+    <item quantity="other">\n      %s Topics in Progress\n    </item>
+  </plurals>
```

Comment:
There are incorrect new lines inserted in these plurals. Tested it out and they definitely display incorectly within the app.